airbyte-integrations/connectors/source-slack/source_slack/schemas/channel_messages.json

```diff
@@ -207,6 +207,17 @@
     },
     "username": {
       "type": ["null", "string"]
+    },
+    "edited": {
+      "properties": {
+        "ts": {
+          "type": ["null", "string"]
+        },
+        "user": {
+          "type": ["null", "string"]
+        }
+      },
+      "type": ["null", "object"]
```

Comment:
Please add `additionalProperties: true` this makes the field backward compatible with new or removed fields in the future.